guides/tutorials/codelabs/kubernetes-v2-source-to-prod/index.md

```diff
@@ -196,12 +196,13 @@ NAME        TYPE       CLUSTER-IP    EXTERNAL-IP   PORT(S)          AGE
 spin-gate   NodePort   10.7.255.85   <none>        8084:31355/TCP   32m
 ```
 
-At this point, you will (for this codelab only) have to open your firewalls on a node
-(with IP `$NODE_IP`) in that cluster to all addresses on for TCP connections on
-`$NODE_PORT`. If (for a production use-case) you were running Spinnaker with
-[authentication](/setup/security), only webhooks would be allowed, which you
-can reject by header or payload. See [the webhook guide for more 
-details](/guides/user/triggers/webhooks).
+Now pick any node in the cluster and record its IP as `$NODE_IP`; for the purposes of
+this codelab, we'll be sending external webhooks to `$NODE_PORT` on that node.  In
```

Comment:
Extraneous space between the sentences.

Reply:
Ha, good catch, my instinct is to always type two spaces.  I'll fix it, but will wait to see if other reviewers have comments.